std/range/package.d

```diff
@@ -5553,6 +5552,12 @@ nothrow @nogc @safe unittest
     auto t1 = iota(0, 10, 2);
     auto t2 = iota(1, 1, 0);
     //float overloads use std.conv.to so can't be @nogc or nothrow
+    alias ssize_t = Unsigned!size_t;
```

Comment:
When adding tests for a bug, it's nice to add a comment mentioning the Bugzilla issue number.